graphene_sqlalchemy/registry.py

```diff
@@ -1,32 +1,89 @@
+from sqlalchemy.types import Enum as SQLAlchemyEnumType
+
+from graphene import Enum
+
+
 class Registry(object):
     def __init__(self):
         self._registry = {}
         self._registry_models = {}
+        self._registry_orm_fields = {}
         self._registry_composites = {}
+        self._registry_enums = {}
+        self._registry_sort_enums = {}
 
-    def register(self, cls):
+    def register(self, obj_type):
         from .types import SQLAlchemyObjectType
 
-        assert issubclass(cls, SQLAlchemyObjectType), (
-            "Only classes of type SQLAlchemyObjectType can be registered, "
-            'received "{}"'
-        ).format(cls.__name__)
-        assert cls._meta.registry == self, "Registry for a Model have to match."
+        if not isinstance(obj_type, type) or not issubclass(
+            obj_type, SQLAlchemyObjectType
+        ):
+            raise TypeError(
+                "Expected SQLAlchemyObjectType, but got: {!r}".format(obj_type)
+            )
+        assert obj_type._meta.registry == self, "Registry for a Model have to match."
         # assert self.get_type_for_model(cls._meta.model) in [None, cls], (
         #     'SQLAlchemy model "{}" already associated with '
         #     'another type "{}".'
         # ).format(cls._meta.model, self._registry[cls._meta.model])
-        self._registry[cls._meta.model] = cls
+        self._registry[obj_type._meta.model] = obj_type
 
     def get_type_for_model(self, model):
         return self._registry.get(model)
 
+    def register_orm_field(self, obj_type, field_name, orm_field):
+        from .types import SQLAlchemyObjectType
+
+        if not isinstance(obj_type, type) or not issubclass(
+            obj_type, SQLAlchemyObjectType
+        ):
+            raise TypeError(
+                "Expected SQLAlchemyObjectType, but got: {!r}".format(obj_type)
+            )
+        if not field_name or not isinstance(field_name, str):
+            raise TypeError("Expected a field name, but got: {!r}".format(field_name))
+        self._registry_orm_fields.setdefault(obj_type, {})[field_name] = orm_field
```

Comment:
Wouldn't it be easier to make `self._registry_orm_fields` a defaultdict? That way every object type has an empty registry.

```suggestion
        self._registry_orm_fields[obj_type][field_name] = orm_field
```